server/src/main/java/org/spine3/server/event/enrich/EventEnricher.java

```diff
@@ -55,16 +55,18 @@
 import static org.spine3.protobuf.Messages.toMessageClass;
 
 /**
- * {@code Enricher} extends information of an event basing on its type and content.
+ * {@code EventEnricher} extends information of an event basing on its type and content.
```

Comment:
Have you included the `EventEnricher`-related changes to this PR by a mistake? They just don't seem related to the `Mismatch`es at all.